### PR TITLE
Add logic to retrieve correct setup phase

### PR DESCRIPTION
**Pull request by os-micmec**
Now that we have the task definition component, we need to incorporate that into retrieving the right setup phase.

Towards #1053

### CODE REVIEW

File: web/src/app/pages/create-survey/create-survey.component.ts [hunk 1]
```diff
@@ -36,13 +38,17 @@ import {ShareSurveyComponent} from './share-survey/share-survey.component';
 export class CreateSurveyComponent implements OnInit {
   currentSurveyId: string | null = null;
   currentSurvey?: Survey;
+  // TODO: when we refresh, the setupPhase below is always displayed for a split of a second.
```
Comment (gino-m) [inline web/src/app/pages/create-survey/create-survey.component.ts:41]: Please add issue number in TODO(#..).

**Review by gino-m — CHANGES_REQUESTED** [review]
LG, just maybe revert the two package-lock.json files which I think got updated accidentally.

File: web/src/app/pages/create-survey/create-survey.component.ts (hunk 1 above)
Comment (gino-m) [reply web/src/app/pages/create-survey/create-survey.component.ts:41]: Gentle ping!

Comment (os-micmec) [reply web/src/app/pages/create-survey/create-survey.component.ts]: Done!

**Review by gino-m — APPROVED** [review]
Thanks for the fix!

**Comment by os-micmec** [comment]
Also removed the package-lock files

**Pull request merged**